packages/backend/src/open_payments/receiver/model.ts

```diff
@@ -35,29 +35,17 @@ export class Receiver {
     incomingPayment: OpenPaymentsIncomingPaymentWithPaymentMethod,
     isLocal: boolean
   ) {
-    if (incomingPayment.completed) {
-      throw new Error('Cannot create receiver from completed incoming payment')
-    }
-
     const expiresAt = incomingPayment.expiresAt
       ? new Date(incomingPayment.expiresAt)
       : undefined
 
-    if (expiresAt && expiresAt.getTime() <= Date.now()) {
-      throw new Error('Cannot create receiver from expired incoming payment')
-    }
-
-    if (!incomingPayment.methods.length) {
-      throw new Error('Missing payment method(s) on incoming payment')
-    }
-
     const incomingAmount = incomingPayment.incomingAmount
       ? parseAmount(incomingPayment.incomingAmount)
       : undefined
     const receivedAmount = parseAmount(incomingPayment.receivedAmount)
 
     // TODO: handle multiple payment methods
-    const ilpMethod = incomingPayment.methods.find(
+    const ilpMethod = incomingPayment.methods?.find(
       (method) => method.type === 'ilp'
     )
     if (!ilpMethod) {
```

Comment:
Maybe we should be a bit more permissive here, and _not_ throw if there are no payment methods provided. (we can keep the check for invalid ilp address - if payment method exists).  What do you think?